lib/method.js

```diff
@@ -16,6 +16,13 @@ module.exports = class Method extends CollectionItem {
       this.parameters = this.getParameters()
       this.returns = this.getReturns()
 
+      const subProp = this.name.split('.')
```

Comment:
This recognizes patterns such as `app.dock.show()` and splits it into just the `show()` name and declares it has a "superObject" called "dock"